components/Explorer/Explorer.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
 import React from 'react';
```

Comment:
It's costing me a lot of time to have to fix the linting on this and I'm still iterating on it rapidly.